docs/core/plugins/mail_crypt.md

```diff
@@ -415,7 +415,7 @@ share the key to groups or someone with no public key.
 
 ## Decrypting Files Encrypted with mail-crypt Plugin
 
-You can use [`decrypt.rb`][decrypt.rb] to decrypt encrypted files.
+You can use [`decrypt.py`][decrypt.py] to decrypt encrypted files.
```

Comment:
We don't use these disconnected links anymore (this looks like leftover from restructured text.

Should be:

```
You can use [`decrypt.rb`](https://github.com/dovecot/tools/blob/main/dcrypt-decrypt.rb)
to decrypt encrypted files.
```